src/main/java/edu/kit/hci/soli/domain/Booking.java

```diff
@@ -14,11 +11,12 @@
 /**
  * The datamodel for a Booking as it is stored in the database
  */
-@Entity
+@Entity(name = "tbl_booking")
```

Comment:
We have a database to ourselves, so we don't really need an application prefix.
If you think we should set one up anyways, `soli` is probably better than `tbl`

Reply:
You are misreading this, this is the name of our table... So spring will generate a table called "tbl_booking" which is contained inside the soli database

Reply:
I know, I just think that calling the table `tbl_booking` is kind of redundant.
With the first line, I wanted to express that, if we shared the database with other apps, I'd agree we need a prefix, but that this doesn't apply.

Reply:
I prefer this to just booking...

Reply:
> If you think we should set one up anyways, `soli` is probably better than `tbl`

Our app is called "Solidarische Raumnutzung", not "Table", which is why I think we should use a prefix based on that. For example, Nextcloud uses `oc`.
That said, other apps (like synapse or reposilite) forego a prefix entirely, which seems reasonable since we don't share the databse with anyone.

Reply:
alright, then its probably better to remove it entirely